### ISSUE TITLE
The sim for shamans is not taking into account the dps/tps from lightning shield procs

### ISSUE THREAD

**Issue by mygamingantics** [wowsims/wotlk#1052]
**Describe the bug**
When choosing lightning shield and setting up as main tank to be hit by the npc, there are no lighting shield procs reported in the detailed results tab.

**To Reproduce**
1. Go to the settings tab
2. Select lighting shield from the player area
3. Select main tank as the tank assignment on the 'other' area
4. Run the dps simulation
5. Click on the detailed results tab
6. Lightning shield is not represented (it will show up if the user selected the static shock talent but not otherwise, and those are only static shock procs)

**Expected behavior**
The user can see the dps/tps from lightning shield if being hit by a mob

**Comment by mygamingantics** [comment]
> f

Yes, that seems to be the case. The static shock procs were being recorded but the lightning shield procs caused by the mob hitting the shaman are not.

**Comment by Wumblesnarf** [comment]
correct, yeah, it is something i want to have in the sim eventually (it is sometimes relevant, patchwerk for example) but its pretty low priority rn
